template_test.go

```diff
@@ -391,3 +391,9 @@ func TestReadYAMLRequestFails(t *testing.T) {
 		})
 	}
 }
+
+func TestReadYAMLSucceeds(t *testing.T) {
+	opts := newOptions()
+	err := readYAMLFile("testdata/valid.yab", nil, opts)
+	assert.NoError(t, err)
+}
```

Comment:
Can we add more test please? We should provide the following unit tests:
- Yab template with file + JSON format
- Yab template with file + YAML format
- Yab template with file + RAW format
- Yab template with file + Proto format
- Yab template with file + Thrift format
- Negative test with bad combinaison of options (like File and request together)